homeassistant/components/light/hue.py

```diff
@@ -27,6 +29,10 @@
 
 PHUE_CONFIG_FILE = "phue.conf"
 
+SUPPORT_HUE = (SUPPORT_BRIGHTNESS | SUPPORT_COLOR_TEMP | SUPPORT_EFFECT |
```

Comment:
The reason for the capabilities field can be found at the discussion at https://github.com/home-assistant/homebridge-homeassistant/pull/23.
In short, when the light is off, the API doesn't give any hints as to what the light supports. Only when it's on it includes all of the relevant attributes. This makes it difficult for consumers of the API to know what to expect.

As for the devices themselves, I had to take educated guesses as to what they support as all I have is my home-made lights. I don't see the Hue component making any differences according to the specific model when setting values. Perhaps there is a way to query the device itself for what it supports in order to have a more fine-grained control (as was done with the MQTT component).